src/tools/rados/rados.cc

```diff
@@ -1317,20 +1325,26 @@ static void dump_inconsistent(const inconsistent_obj_t& inc,
   f.open_object_section("object");
   dump_object_id(inc.object, f);
   f.close_section();
+
+  f.open_array_section("errors");
+  if (inc.has_attr_unexpected())
+    f.dump_string("error", "attr_unexpected");
   if (inc.has_shard_missing())
-    f.dump_bool("missing", inc.has_shard_missing());
+    f.dump_string("error", "missing");
   if (inc.has_stat_error())
-    f.dump_bool("stat_err", inc.has_stat_error());
+    f.dump_string("error", "stat_err");
```

Comment:
and change "stat_err" to "stat_error", so it is consistent with "read_error" below.